api_ekyc.go

```diff
@@ -50,69 +50,125 @@ type ResponseEkycLiveness struct {
 	Err  *Error            `json:"error"`
 }
 
-func (c Client) EkycMyKad(request RequestEkycMykad) (*ResponseEkycMykad, error) {
+type ResponseGetEkycResult struct {
+	Code string        `json:"code"`
+	Item GetEkycResult `json:"item"`
+	Err  *Error        `json:"error"`
+}
+
+// EkycMyKad :
+func (c Client) EkycMyKad(request RequestEkycMykad) (*ResponseEkycMykad, *Error) {
 	if c.err != nil {
-		return nil, c.err
+		return nil, &Error{
+			Code:    "INTERNAL_SERVER_ERROR",
+			Message: c.err.Error(),
+		}
 	}
 
 	response := new(ResponseEkycMykad)
```

Comment:
I think this can put after line `80`, don't allocate variable too early if you not using it, the GC needs to cleanup